src/components/avatar/__tests__/index.spec.js

```diff
@@ -58,4 +59,40 @@ describe('Avatar Badge', () => {
       expect(uut.renderBadge()).toEqual(undefined);
     });
   });
+
+  describe('badgeProps.size, supports enum or number', () => {
+    it('should return 10 as the size number given', () => {
+      const uut = new Avatar({badgeProps: {size: 10}});
+      expect(uut.getBadgeSize()).toEqual(10);
+    });
+
+    it('should return 876 as the size number given', () => {
+      const uut = new Avatar({badgeProps: {size: 876}});
+      expect(uut.getBadgeSize()).toEqual(876);
+    });
+
+    it('should return 0 as the given size number', () => {
+      const uut = new Avatar({badgeProps: {size: 0}});
+      expect(uut.getBadgeSize()).toEqual(0);
+    });
+
```

Comment:
Not the end of the world obviously but having 3 tests testing numeric input might be 2 tests too much 😸 .
If one of these tests fail, all of them will fail for the same reason.
For your consideration...

Reply:
right, keeping two for zero/undefined check + some number